config/entries/alert.txt

```diff
@@ -1,11 +1,11 @@
 ## ALERT LEVELS ###
-ALERT_GREEN All threats to the station have passed. Security may not have weapons visible, privacy laws are once again fully enforced.
-ALERT_BLUE_UPTO The station has received reliable information about potential threats to the station. Security staff may have weapons visible, random searches are permitted.
-ALERT_BLUE_DOWNTO Significant confirmed threats have been neutralized. Security may no longer have weapons drawn at all times, but may continue to have them visible. Random searches are still permitted.
-ALERT_AMBER_UPTO There are signficant confirmed threats to the station. Security staff may have weapons unholstered at all times. Random searches are allowed and advised.
-ALERT_AMBER_DOWNTO The immediate threat has passed. Security is no longer authorized to use lethal force, but may continue to have weapons drawn. Access requirements have been restored.
-ALERT_RED_UPTO There is an immediate serious threat to the station. Security is now authorized to use lethal force. Additionally, access requirements on some machines have been lifted.
-ALERT_RED_DOWNTO The station's destruction has been averted. There is still however an immediate serious threat to the station. Security is still authorized to use lethal force.
+ALERT_GREEN All threats to the station have passed.
+ALERT_BLUE_UPTO The station has received reliable information about potential threats to the station.
+ALERT_BLUE_DOWNTO Significant confirmed threats have been neutralized.
+ALERT_AMBER_UPTO There are signficant confirmed threats to the station.
+ALERT_AMBER_DOWNTO The immediate threat has passed.
+ALERT_RED_UPTO There is an immediate serious threat to the station.
+ALERT_RED_DOWNTO The station's destruction has been averted. There is still however an immediate serious threat to the station.
```

Comment:
Since these are config changes they'd probably be better made directly on the local config